pkgs/top-level/all-packages.nix

```diff
@@ -7095,7 +7095,7 @@ in
     trilium-desktop
     trilium-server
     ;
-  trilium = trilium-desktop;
+  trilium = lib.warn ''The 'trilium' package is deprecated. Use 'trilium-desktop' instead.''  trilium-desktop;
```

Comment:
I think we do deprecations in `top-level/aliases.nix`, e.g.: https://github.com/NixOS/nixpkgs/blob/7610f0417b2f3c42f99ff1f30f8224007fb31a20/pkgs/top-level/aliases.nix#L59

Reply:
Done